### DISCUSSION TITLE
HomeKit new architecture

### DISCUSSION THREAD

**Discussion by kakiser56** [homebridge-simplisafe3/homebridge-simplisafe3#393] [General]
Wanted to know if the new HomeKit architecture will break the plugin? I really love this plugin, and don't want to make it unusable!

Thanks

**Comment by ibmaster** [comment]
Working just fine for me under the newest HomeKit architecture.Sent from my iPhoneOn Jun 19, 2023, at 8:20 AM, kakiser56 ***@***.***> wrote:﻿
Wanted to know if the new HomeKit architecture will break the plugin? I really love this plugin, and don't want to make it unusable!
Thanks

**Comment by shamoon** [comment]
Yes, should have no issues. The entire idea of the new architecture is that the API (which is what Homebridge uses) hasn't changed
